### ISSUE TITLE
UK vs US spelling

### ISSUE THREAD

**Issue by bianchini88** [ELIXIR-NO/website#35]
We should decide on this. The landing page currently contains terms from both standards (we write e.g. "standardise" in a caption and "specialized" in another section).
@jonlaerdahl @ingeborgwinge. Note that this decision should be communicated to all the editors (who would ideally check for compliance when reviewing a PR).

**Comment by ingeborgwinge** [comment]
@bianchini88 Did you discuss this already? I agree that we should have that standardised. I am more for british, but that is a personal preference :-)

**Comment by bianchini88** [comment]
@ingeborgwinge No, this was not discussed. UK spelling is also my personal preference, and I think it would make sense to use it to match ELIXIR Europe's website and other related ELIXIR-branded resources (e.g. RDMkit). I think a decision on spelling should come from above. If you think we should go for UK spelling, it is enough for me to enforce this. Or you can have a quick chat with Sushma and/or the other node leaders and let me know.

**Comment by ingeborgwinge** [comment]
I am pretty sure that Sushma will agree, but I will ask her. I'll let you know.
And I am sorry that I haven't been so active here... I tend to forget to look here for new messages...

**Comment by bianchini88** [comment]
Decision: UK spelling. @bianchini88 will keep an eye on this as we keep on working on the content.
